### DISCUSSION TITLE
The Assimp 5.2.5 Bugfix-Release

### DISCUSSION THREAD

**Discussion by kimkulling** [assimp/assimp#4718] [General]
## What's Changed
* Add unittest to reproduce undefined behavior by @kimkulling in https://github.com/assimp/assimp/pull/4532
* Cleanups by @kimkulling in https://github.com/assimp/assimp/pull/4537
* Link static linkage for std_image. by @kimkulling in https://github.com/assimp/assimp/pull/4478
* fix msvc warnings-as-errors by @Gargaj in https://github.com/assimp/assimp/pull/4549
* Remove dll-export tags from static library builds. by @Underdisc in https://github.com/assimp/assimp/pull/4555
* Fix Import a specific FBX model will freeze the app #4395 by @Nor-s in https://github.com/assimp/assimp/pull/4554
* Create SECURITY.md by @kimkulling in https://github.com/assimp/assimp/pull/4565
* Pragma warnings cause build fail with MinGW by @ethaninfinity in https://github.com/assimp/assimp/pull/4564
* Fixed FBXConverter build error when using double precision by @matthewclendening in https://github.com/assimp/assimp/pull/4546
* Fix  possible nullptr exception by @kimkulling in https://github.com/assimp/assimp/pull/4567
* [Experimental] New skeleton container for bones by @kimkulling in https://github.com/assimp/assimp/pull/4552
* Add support for GCC v12 by @PercentBoat4164 in https://github.com/assimp/assimp/pull/4578
* Remove unused variable. by @kovacsv in https://github.com/assimp/assimp/pull/4584
* Infinite loop on bad import files by @tanolino in https://github.com/assimp/assimp/pull/4534
* Utilize AI_CONFIG_IMPORT_REMOVE_EMPTY_BONES flag for Collada meshes. by @onurtore in https://github.com/assimp/assimp/pull/4585
* Fix Windows 32-bit builds by @Gargaj in https://github.com/assimp/assimp/pull/4581
* Fix GNUC check on Windows by @hgdagon in https://github.com/assimp/assimp/pull/4542
* Update the name of the package by @kimkulling in https://github.com/assimp/assimp/pull/4595
* Kimkulling/fix invalid opengex token match by @kimkulling in https://github.com/assimp/assimp/pull/4596
* Disable build for tools per default by @kimkulling in https://github.com/assimp/assimp/pull/4598
* Use mingw.include by @kimkulling in https://github.com/assimp/assimp/pull/4601
* Fix a memory leak by @kimkulling in https://github.com/assimp/assimp/pull/4605
* Try to fix issue 4238 by @kimkulling in https://github.com/assimp/assimp/pull/4609
* Fix nested animation name being lost in Collada by @luca-della-vedova in https://github.com/assimp/assimp/pull/4597
* chore: Included GitHub actions in the dependabot config by @naveensrinivasan in https://github.com/assimp/assimp/pull/4569
* Fix possible bad_alloc exception for an invalid file by @kimkulling in https://github.com/assimp/assimp/pull/4614
* Bump JesseTG/rm from 1.0.2 to 1.0.3 by @dependabot in https://github.com/assimp/assimp/pull/4613
* Bump actions/cache from 2 to 3 by @dependabot in https://github.com/assimp/assimp/pull/4612
* Kimkulling/fix texture loading 3MF, reladed issue-4568 by @kimkulling in https://github.com/assimp/assimp/pull/4619
* Bump actions/upload-artifact from 2 to 3 by @dependabot in https://github.com/assimp/assimp/pull/4610
* Bump actions/checkout from 2 to 3 by @dependabot in https://github.com/assimp/assimp/pull/4611
* I ran into an error while processing colored binary stl. Just a type but it better be fixed by @blackhorse-reddog in https://github.com/assimp/assimp/pull/4541
* Remove assertion test by @kimkulling in https://github.com/assimp/assimp/pull/4627
* Fix memory leak in D3MFOpcPackage by @kimkulling in https://github.com/assimp/assimp/pull/4629
* Fix typo in installation instructions for ubuntu. by @hectorpiteau in https://github.com/assimp/assimp/pull/4621
* Build fix for compiling against minizip. by @robertosfield in https://github.com/assimp/assimp/pull/4631
* Fix stl for over 4 GB by @tanolino in https://github.com/assimp/assimp/pull/4630
* Fix uninitialized variable. by @kimkulling in https://github.com/assimp/assimp/pull/4642
* Fixes Crash in Assimp::ObjFileMtlImporter::getFloatValue by @sashashura in https://github.com/assimp/assimp/pull/4647
* Fixes Heap-buffer-overflow in Assimp::ObjFileParser::getFace by @sashashura in https://github.com/assimp/assimp/pull/4646
* Fixes Heap-buffer-overflow in std::__1::basic_string<char, std::__1::… by @sashashura in https://github.com/assimp/assimp/pull/4645
* Fixes Heap-use-after-free in Assimp::DXFImporter::ExpandBlockReferences by @sashashura in https://github.com/assimp/assimp/pull/4644
* Fixes Heap-buffer-overflow in SuperFastHash by @sashashura in https://github.com/assimp/assimp/pull/4643
* ColladaParser - Store sid in mSID field by @luca-della-vedova in https://github.com/assimp/assimp/pull/4538
* Fix mingw include in assimp_cmd.rc by @Koekto-code in https://github.com/assimp/assimp/pull/4635
* Fix warnings that are causing build fails with specific build flags by @enginmanap in https://github.com/assimp/assimp/pull/4632
* Update version tag by @waebbl in https://github.com/assimp/assimp/pull/4656
* Improvements and optimizations for the obj-parsers. by @kimkulling in https://github.com/assimp/assimp/pull/4666
* Experiment: try to enable parallel build by @kimkulling in https://github.com/assimp/assimp/pull/4669
* Fixed typo by @Fiskmans in https://github.com/assimp/assimp/pull/4668
* Use  [[fallthrough]]; to mark whished fallthroughs by @kimkulling in https://github.com/assimp/assimp/pull/4673
* Kimkulling/do not add dot when the extension is empty issue 4670 by @kimkulling in https://github.com/assimp/assimp/pull/4674
* Fixes Heap-buffer-overflow READ in Assimp::ASE::Parser::ParseLV1SoftSkinBlock by @sashashura in https://github.com/assimp/assimp/pull/4680
* Use unqualified uint32_t everywhere in FBXBinaryTokenizer by @villevoutilainen in https://github.com/assimp/assimp/pull/4678
* Fix problems setting DirectX_LIBRARY by @Koekto-code in https://github.com/assimp/assimp/pull/4681
* Added support for more bone weights in GLTF2  by @Promit in https://github.com/assimp/assimp/pull/4453
* (Mostly) Blender fixes by @turol in https://github.com/assimp/assimp/pull/4679
* [WIP] Use ai_Real to write correct accuracy by @kimkulling in https://github.com/assimp/assimp/pull/4697
* SMD fixes by @turol in https://github.com/assimp/assimp/pull/4699
* Remove exception on glTF 2.0 loading by @vkaytsanov in https://github.com/assimp/assimp/pull/4693
* Fix out-of-bounds reads in X3D importer by @turol in https://github.com/assimp/assimp/pull/4701
* Apply the modernize-use-emplace clang-tidy rule by @Skylion007 in https://github.com/assimp/assimp/pull/4700
* The Wrong object is created here! by @JG-Adams in https://github.com/assimp/assimp/pull/4704
* [WIP] Code cleanup and some new unittests for edge-cases. by @kimkulling in https://github.com/assimp/assimp/pull/4705
* clang-tidy: explicitly default all empty ctors and dtors by @Skylion007 in https://github.com/assimp/assimp/pull/4703
* fix vertices being joined duplicating weights by @Gargaj in https://github.com/assimp/assimp/pull/4707
* add missing light data to assbin import/export by @Gargaj in https://github.com/assimp/assimp/pull/4702
* Fix aiBone.mOffsetMatrix documentation by @ChrisBlueStone in https://github.com/assimp/assimp/pull/4713
* Minor obj export bugfix by @HiMemX in https://github.com/assimp/assimp/pull/4716
* Kimkulling/cleanup after reviewing by @kimkulling in https://github.com/assimp/assimp/pull/4715

## New Contributors
* @Underdisc made their first contribution in https://github.com/assimp/assimp/pull/4555
* @Nor-s made their first contribution in https://github.com/assimp/assimp/pull/4554
* @ethaninfinity made their first contribution in https://github.com/assimp/assimp/pull/4564
* @matthewclendening made their first contribution in https://github.com/assimp/assimp/pull/4546
* @PercentBoat4164 made their first contribution in https://github.com/assimp/assimp/pull/4578
* @onurtore made their first contribution in https://github.com/assimp/assimp/pull/4585
* @luca-della-vedova made their first contribution in https://github.com/assimp/assimp/pull/4597
* @naveensrinivasan made their first contribution in https://github.com/assimp/assimp/pull/4569
* @dependabot made their first contribution in https://github.com/assimp/assimp/pull/4613
* @blackhorse-reddog made their first contribution in https://github.com/assimp/assimp/pull/4541
* @hectorpiteau made their first contribution in https://github.com/assimp/assimp/pull/4621
* @robertosfield made their first contribution in https://github.com/assimp/assimp/pull/4631
* @sashashura made their first contribution in https://github.com/assimp/assimp/pull/4647
* @Koekto-code made their first contribution in https://github.com/assimp/assimp/pull/4635
* @enginmanap made their first contribution in https://github.com/assimp/assimp/pull/4632
* @waebbl made their first contribution in https://github.com/assimp/assimp/pull/4656
* @Fiskmans made their first contribution in https://github.com/assimp/assimp/pull/4668
* @vkaytsanov made their first contribution in https://github.com/assimp/assimp/pull/4693
* @JG-Adams made their first contribution in https://github.com/assimp/assimp/pull/4704
* @ChrisBlueStone made their first contribution in https://github.com/assimp/assimp/pull/4713
* @HiMemX made their first contribution in https://github.com/assimp/assimp/pull/4716

**Full Changelog**: https://github.com/assimp/assimp/compare/v5.2.4...v5.2.5

<hr /><em>This discussion was created from the release <a href='https://github.com/assimp/assimp/releases/tag/v5.2.5'>The Assimp 5.2.5 Bugfix-Release</a>.</em>

**Comment by kimkulling** [comment]
Version was not updated, again ...

**Reply by turol** [reply]
Maybe you should update the version number immediately after a release so the work-in-progress has the new version already.
